### DISCUSSION TITLE
In 08_pytorch_paper_replicating. What is AK Twitter account? I can't find it.

### DISCUSSION THREAD

**Discussion by BroswerChou** [mrdbourke/pytorch-deep-learning#1145] [Q&A]
![image](https://github.com/user-attachments/assets/75daf12f-f283-455f-bc51-f7c2a06b3f80)

**Comment by pem131** [comment]
Greetings BrowserChou,

Twitter is now known as X.

Try to search for @_akhaliq on X and you should find the account.

Good luck.
